src/video/gl_backend.c

```diff
@@ -36,9 +37,13 @@ enum shader_attr {
   ATTR_IGNORE_ALPHA = 0x8,
   ATTR_IGNORE_TEXTURE_ALPHA = 0x10,
   ATTR_OFFSET_COLOR = 0x20,
-  ATTR_COUNT = 0x40,
+  ATTR_PUNCH_THROUGH = 0x40,
```

Comment:
I'd match this to the changes mentioned above by calling it `ATTR_PT_ALPHA_TEST`.